typescript/lambda-api-ci/lib/lambda-api-stack.ts

```diff
@@ -0,0 +1,58 @@
+import {
+  AuthorizationType,
+  LambdaIntegration,
+  MethodLoggingLevel,
+  RestApi
+} from "@aws-cdk/aws-apigateway"
+import { PolicyStatement } from "@aws-cdk/aws-iam"
+import { Function, Runtime, AssetCode, Code } from "@aws-cdk/aws-lambda"
+import {Construct, Duration, Stack, StackProps} from "@aws-cdk/core"
+import s3 = require("@aws-cdk/aws-s3");
+
+interface LambdaApiStackProps extends StackProps{
+  functionName: string
+}
+
+export class LambdaApiStack extends Stack {
+  private restApi: RestApi
+  private lambdaFunction: Function
+  private bucket: s3.Bucket
+
+  constructor(scope: Construct, id: string, props: LambdaApiStackProps) {
+    super(scope, id, props);
+
+    this.bucket = new s3.Bucket(this, "WidgetStore");
+
+    this.restApi = new RestApi(this, "RestApi", {
+      deployOptions: {
+        stageName: "beta",
+        metricsEnabled: true,
+        loggingLevel: MethodLoggingLevel.INFO,
+        dataTraceEnabled: true
+      }
+    })
+
+    const lambdaPolicy = new PolicyStatement();
```

Comment:
Please be consistent with semicolons. We recommend all lines terminate with semicolon, but if you want to follow the no-semicolon TS guidance, stick to that throughout the example.